packages/nimble-components/src/table/testing/table.pageobject.ts

```diff
@@ -526,39 +528,44 @@ export class TablePageObject<T extends TableRecord> {
         }
     }
 
+    /**
+     * @param columnIndex The index of the column to the left of the divider to press.
+     */
+    public pressRightColumnDivider(columnIndex: number): void {
+        this.pressColumnDivider(this.getColumnDivider(columnIndex, false));
+    }
+
+    /**
+     * @param columnIndex The index of the column to the right of the divider to press.
+     */
+    public pressLeftColumnDivider(columnIndex: number): void {
```

Comment:
Why aren't we using any of the left divider page object APIs?

Reply:
Looks like we still have a couple tests using `dragSizeColumnByLeftDivider`, but you're right--we're not using the `press/releaseLeftColumnDivider` functions I added. The right/left distinction is really only useful from a API ease of use perspective, since there aren't actually separate "left" and "right" dividers. I.e. the left divider of column 2 is the same element as the right divider of column 1. Arguably, we don't need separate left/right versions of the drag-size-column functions either, but then you'd have to identify the divider by index, which is (maybe?) a little harder to understand than, say, "the left divider of column three."

I'll remove the unused functions.